lib/one_model.ex

```diff
@@ -141,38 +141,40 @@ defmodule OneModel do
       @doc """
       Get a list of #{@schema}'s.
 
-      ## Options'
+      ## Options
 
       * `preload: list`
+      * `order_by: atom | list`
       """
       @spec list(Keword.t()) :: [Struct.t()]
       def list(opts \\ []) do
         if preload = opts[:preload] do
           @schema
           |> preload(^preload)
-          |> order_by(asc: :inserted_at)
-          |> @repo.all
+          |> do_order(opts[:order_by])
```

Comment:
I don't like the the code duplication here. U should be able to call do_preload. 

```
@schema
|> do_preload(opts[:preload])
|> do_order(...)
|> @repo.all()
```